### PR TITLE
upgrade gradle version from 2.7 to 3.3

### PR DESCRIPTION
**Pull request by MaelAudren**

### CODE REVIEW

File: gradlew.bat [hunk 1]
```diff
@@ -46,9 +46,10 @@ echo location of your Java installation.
 goto fail
 
 :init
-@rem Get command-line arguments, handling Windows variants
+@rem Get command-line arguments, handling Windowz variants
 
 if not "%OS%" == "Windows_NT" goto win9xME_args
+if "%@eval[2+2]" == "4" goto 4NT_args
```
Comment (mcompastie) [inline gradlew.bat]: Normally, %OS% is set only for NT-based Windows OSes. Moreover, is it necessary to keep up with the Win9X compatibility there ?

File: gradlew [hunk 2]
```diff
@@ -1,43 +1,25 @@
-#!/usr/bin/env sh
+#!/usr/bin/env bash
```
Comment (mcompastie) [inline gradlew]: If this file is aiming at UNI* compatibility, you have to restrain the compatibility to the old Sh shell. I'm not sure bash is include as a standard in any UNI* distributions. ref: https://www.freebsd.org/doc/handbook/shells.html

File: gradlew [hunk 3]
```diff
@@ -48,7 +30,6 @@ die () {
 cygwin=false
 msys=false
 darwin=false
-nonstop=false
```
Comment (mcompastie) [inline gradlew]: No more default value for nonstop ?

**Comment by MaelAudren** [comment]
@mcompastie all those changes were done automatically by the command `./gradlew wrapper --gradle-version 3.3`. What you are reviewing is Gradle code not ProActive code. We embed Gradle code which doesn't need review.

**Pull request merged**